frontend/package.json

```diff
@@ -104,13 +104,15 @@
         "@stripe/stripe-js": "^4.5.0",
         "@tailwindcss/cli": "4.0.11",
         "@testing-library/dom": ">=7.21.4",
-        "@tiptap/core": "^2.1.16",
+        "@tiptap/core": "^2.26.1",
         "@tiptap/extension-document": "^2.1.16",
         "@tiptap/extension-floating-menu": "^2.1.16",
+        "@tiptap/extension-link": "^3.0.7",
```

Comment:
Do we not already support links in Notebooks?
<img width="434" height="58" alt="Screenshot 2025-08-05 at 10 52 41" src="https://github.com/user-attachments/assets/0ac855ad-ba3d-4483-8729-66546fbe24e5" />

Reply:
Yes I'm dumb and didn't see that. I'm removing it and just keep the underline plugin.

Reply:
No worries, otherwise looked good 👌